### ISSUE TITLE
Add row actions menu to Project Members table

### ISSUE THREAD

**Issue by jewharton** [storj/storj#5762]
## Motivation
We want our UI to be consistent with our updated designs.

Blocked by #5353 and #5751 

## Acceptance Criteria
Each row of the project members table contains an ellipses button that, when clicked, displays a dropdown with the following options:
* The "Resend invite" option resends an invitation to the row's user. This only appears once the existing invitation has expired.
  * add Segment event: `resend_invite_clicked`
* The "Copy invite" option copies the invitation link sent to the row's user. This only appears for pending project members and does not appear if the invitation has expired.
* The "Remove member" option begins the process of removing the row's user from the project by displaying the member removal modal.
  * add Segment event: `remove_member_clicked`

## Out of Scope
* Implementation of the member removal modal (#5751)

## Figma Designs
https://www.figma.com/file/HlmasFJNHxs2lzGerq3WYH/Satellite-GUI?node-id=8504%3A18815
